src/inventory/gilded_rose.js

```diff
@@ -21,59 +21,99 @@ const items = [
 
 updateQuality(items);
 */
+function checkNotSpecialItem(item) {
+  return (item.name != 'Aged Brie' &&
+    item.name != 'Backstage passes to a TAFKAL80ETC concert' &&
+    item.name != 'Sulfuras, Hand of Ragnaros')
+};
+function decreaseQuality(item, amount = 1) {
+  item.quality -= amount;
+}
+function increaseQuality(item, amount = 1) {
+  item.quality += amount;
+}
+function decreaseSellIn(item, amount = 1) {
+  item.sell_in -= 1;
+}
+function standardItemChanges(item) {
+  // then check that the quality between 0-50, and decrease the quality by 1
+  if (item.quality > 0 && item.quality < 50) {
+    decreaseQuality(item, 1);
+    // if the sell_in becomes negative, decrease quality again by 1
+    if(item.sell_in < 0) {
+      decreaseQuality(item, 1);
+    }
+  }
+}
+function agedBrieQuality(item) {
+  const { quality } = item;
+  switch (true) {
```

Comment:
Because for both of these cases you check that quality is in between 0 and 50. Your predicate could check for that rather than just being true. 
```suggestion
  switch (quality > 0 && quality < 50) {
```